python/src/ContinuousPC_doc.i.in

```diff
@@ -102,38 +143,10 @@ junction_tree : :py:class:`pyAgrum.JunctionTree`
 
 // ----------------------------------------------------------------------------
 
-%feature("docstring") OTAGRUM::ContinuousPC::PDAGtoDot
-"Convert PDAG graph to dot.
-
-Parameters
-----------
-pdag : :py:class:`pyAgrum.MixedGraph`
-    dag
-
-Returns
--------
-dot : str
-    Dot format representation"
-
-// ----------------------------------------------------------------------------
-
-%feature("docstring") OTAGRUM::ContinuousPC::skeletonToDot
-"Convert skeleton graph to dot.
-
-Parameters
-----------
-pdag : :py:class:`pyAgrum.UndiGraph`
-    dag
 
-Returns
--------
-dot : str
-    Dot format representation"
-
-// ----------------------------------------------------------------------------
 
 %feature("docstring") OTAGRUM::ContinuousPC::getPValue
-"P-Value accessor.
+"P-Value accessor
```

Comment:
I think we need the docstring short description to end with a dot, see pep-0257:
https://www.python.org/dev/peps/pep-0257/#id16